website/src/components/logo-cloud.js

```diff
@@ -49,6 +49,21 @@ export const LogoCloud = () => {
           />
         </HoverLogo>
       </Logo>
+      <Logo to="https://www.tiktok.com/">
```

Comment:
How do you know TikTok is using jotai?

Reply:
Somebody told us, and we confirmed it in the JS code at that time.